packages/interface-internal/src/address-manager.ts

```diff
@@ -57,49 +57,70 @@ export interface ConfirmAddressOptions {
   type?: AddressType
 }
 
+/**
+ * The `AddressManager` module provides an interface for managing peer addresses
+ * in libp2p. It supports handling multiple types of addresses, verifying their validity,
+ * and storing mappings between internal and external addresses.
+ */
+/**
+ * Interface for managing peer addresses in libp2p.
+ */
+
 export interface AddressManager {
```

Comment:
This needs to be one comment, not two, otherwise only the second will appear in the generated docs.

You can check this by running `npm run docs` in the root of the repo and looking at the HTML that appears in the `.docs` folder.

```suggestion
/**
 * The `AddressManager` provides an interface for managing peer addresses
 * in libp2p. It supports handling multiple types of addresses, verifying their validity,
 * and storing mappings between internal and external addresses.
 */
export interface AddressManager {
```